amoro-ams/src/main/java/org/apache/amoro/server/dashboard/controller/OptimizerGroupController.java

```diff
@@ -58,13 +62,24 @@ public OptimizerGroupController(
     this.optimizerManager = optimizerManager;
   }
 
+  public void getActions(Context ctx) {
+    ctx.json(
+        OkResponse.of(
+            Arrays.stream(OptimizingStatus.values())
+                .map(OptimizingStatus::displayValue)
+                .collect(Collectors.toList())));
+  }
+
   /** Get optimize tables. * @return List of {@link TableOptimizingInfo} */
   public void getOptimizerTables(Context ctx) {
     String optimizerGroup = ctx.pathParam("optimizerGroup");
     String dbFilterStr = ctx.queryParam("dbSearchInput");
     String tableFilterStr = ctx.queryParam("tableSearchInput");
+    String[] requestedActions = ctx.queryParamAsClass("actions", String[].class).getOrDefault(null);
```

Comment:
I got following information when trying to ran this PR in my local environment, It seems Javalin cannot parse string array param by default.

```
[JettyServerThreadPool-44] [io.javalin.Javalin] [] - Can't convert to String[]. Register a converter using JavalinValidation#register.
```

Reply:
Sorry for that.

I have tested the new code, PTAL